src/app/app.component.ts

```diff
@@ -41,6 +42,7 @@ export class AppComponent {
   protected subscriptions: Subscription[] = [];
 
   @ViewChild('connectToGithubModal') connectToGithubModal: TemplateRef<any>;
+  @ViewChild('modalAddAppOverlay') modalAddAppOverlay: ModalDirective;
```

Comment:
Along with the changes in the html, this should NOT be here either. It belongs in a child.